payments-core/src/main/java/com/stripe/android/model/PaymentIntentLinkAccountSessionParams.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.model
+
+data class PaymentIntentLinkAccountSessionParams(
```

Comment:
Do we need a SetupIntentLinkAccountSessionParams?

Reply:
yes, I'll commit those in a bit.

Reply:
Actually params for the request should be the same, added setup endpoint and tests to verify urls and params are correct - https://github.com/stripe/stripe-android/pull/4711/commits/48d2d88af728927986c9a6dbb5fd96ffea5ce721